docs/search_syntax.md

```diff
@@ -0,0 +1,228 @@
+# Search Syntax Documentation
+
+## Introduction
+
+This document provides an overview of the search syntax that users can utilize to filter metrics and runs.
+
+## Search Runs
+You can filter the runs using the following run attributes:
+| Property    | Description | Type |
+| -------- | ------- | ------- |
+| ```name```  | Run name | ```string```|
+| ```hash``` | Run hash | ```string```|
+| ```experiment``` | Experiment name | ```string```|
+| ```tags``` | List of run tags |```dictionary```|
+| ```archived``` | True if run is archived, otherwise False | ```boolean```|
+| ```active``` | True if run is active(in progress), otherwise False | ```boolean``` |
+| ```duration``` | Run duration in seconds | ```numeric```|
+| ```created_at``` | Run creation datetime | ```numeric``` |
+| ```finalized_at``` | Run end datetime | ```numeric``` |
+| ```metrics``` | Set of run metrics | ```dictionary``` |
```

Comment:
can these be given table-like format in the markdown source, too? ie, lining up the columns